skiplang/prelude/src/core/primitives/String.sk

```diff
@@ -552,23 +552,23 @@ private fun isWhitespace(c: Char): Bool {
   }
 }
 
-private mutable class StringBytes(
```

Comment:
I think we can also make `ArrayBytes` immutable with the current design (separate `Bytes` vs `MutableBytes`).

Reply:
Ah ah, not really `ArrayBytes` work on `readonly Array`, so it has to be `mutable`, though only `readonly` instances are used